src/App.js

```diff
@@ -1,25 +1,68 @@
-import logo from './logo.svg';
+import React from 'react'
 import './App.css';
+import contacts from './contacts.json'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    initial: contacts.slice(0, 5),
+  }
+
+  randomUser = () => {
+    this.state.initial.push(contacts[Math.floor(Math.random() * contacts.length)]);
+
+    this.setState(this.state.initial)
+  };
+
+  sortByName = () => {
+    this.state.initial.sort((a, b) => {
+      return a.name < b.name ?  -1 : a.name > b.name ? 1 : 0;
+    })
+
+    this.setState(this.state.initial)
```

Comment:
Esto se podría hacer en una línea y se podría hacer así: `this.setState({initial: this.state.initial.sort((a, b) => a.name.localeCompare(b.name)
    })`